src/main/java/com/minecolonies/coremod/commands/colonycommands/DoRaidTonightCommand.java

```diff
@@ -44,31 +52,50 @@ public String getCommandUsage(@NotNull final ICommandSender sender)
     }
 
     @Override
-    public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
+    public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final List<ActionArgument> actionArgumentList,
+            @NotNull final Map<String, Object> argumentValueByActionArgumentNameMap) throws CommandException
     {
-        if (sender instanceof EntityPlayer && !isPlayerOpped(sender))
+        Colony colony = null;
+        final Object colonyObject = argumentValueByActionArgumentNameMap.get("colony");
+        if (null != colonyObject)
         {
-            sender.sendMessage(new TextComponentString("Must be OP to use command"));
-            return;
+            colony = (Colony) colonyObject;
         }
 
+        executeShared(server, sender, colony);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') "NullPointerException" will be thrown when invoking method "executeShared()". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2259)